src/android/android-wizard.ts

```diff
@@ -0,0 +1,150 @@
+    );
+    await abort();
+    return;
+  }
+
+  const appFile = await traceStep('Select App File', () =>
+    gradle.selectAppFile(buildGradleFiles),
+  );
+
+  // ======== STEP 1. Add Sentry Gradle Plugin to build.gradle(.kts) ============
+  clack.log.info(
+    `Adding ${chalk.bold(
+      'Sentry Gradle plugin',
+    )} to your app's build.gradle file.`,
+  );
+  const pluginAdded = await traceStep('Add Gradle Plugin', () =>
+    gradle.addGradlePlugin(appFile),
+  );
+  if (!pluginAdded) {
+    clack.log.warn(
+      "Could not add Sentry Gradle plugin to your app's build.gradle file. You'll have to add it manually.\nPlease follow the instructions at https://docs.sentry.io/platforms/android/#install",
+    );
+  }
+
+  const { selectedProject } = await getOrAskForProjectData(options, 'android');
+
+  // ======== STEP 2. Configure Sentry SDK via AndroidManifest ============
+  clack.log.info(
+    `Configuring Sentry SDK via ${chalk.bold('AndroidManifest.xml')}`,
+  );
+  const buildFile = path.basename(appFile);
+  const appDir = appFile.split(buildFile)[0];
+  const manifestFile = path.join(appDir, 'src', 'main', 'AndroidManifest.xml');
+
+  const manifestUpdated = traceStep('Update Android Manifest', () =>
+    manifest.addManifestSnippet(
+      manifestFile,
+      selectedProject.keys[0].dsn.public,
+    ),
+  );
+  if (!manifestUpdated) {
+    clack.log.warn(
+      "Could not configure the Sentry SDK. You'll have to do it manually.\nPlease follow the instructions at https://docs.sentry.io/platforms/android/#configure",
+    );
+  }
+
+  // ======== STEP 3. Patch Main Activity with a test error snippet ============
+  clack.log.info(
+    `Patching ${chalk.bold('Main Activity')} with a test error snippet.`,
+  );
+  const mainActivity = traceStep('Find Main Activity', () =>
+    manifest.getMainActivity(manifestFile),
+  );
+  let packageName = mainActivity.packageName;
+  if (!packageName) {
+    // if no package name in AndroidManifest, look into gradle script
+    packageName = gradle.getNamespace(appFile);
+  }
+  const activityName = mainActivity.activityName;
+  if (!activityName || !packageName) {
+    clack.log.warn(
+      "Could not find Activity with intent action MAIN. You'll have to manually verify the setup.\nPlease follow the instructions at https://docs.sentry.io/platforms/android/#verify",
+    );
+  } else {
+    const packageNameStable = packageName;
+    const activityFile = traceStep('Find Main Activity Source File', () =>
+      codetools.findActivitySourceFile(appDir, packageNameStable, activityName),
+    );
+
+    const activityPatched = traceStep('Patch Main Activity', () =>
+      codetools.patchMainActivity(activityFile),
+    );
+    if (!activityPatched) {
+      clack.log.warn(
+        "Could not patch main activity. You'll have to manually verify the setup.\nPlease follow the instructions at https://docs.sentry.io/platforms/android/#verify",
+      );
+    }
+  }
```

Comment:
l: I'd recommend adding an outro message to the wizard and a small sentence how to verify the SDK setup. Example from the sveltekit wizard:

```
clack.outro(`
${chalk.green('Successfully installed the Sentry SvelteKit SDK!')}

You can validate your setup by starting your dev environment (`npm run dev`) and visiting "/sentry-example".

Check out the SDK documentation for further configuration:
https://docs.sentry.io/platforms/javascript/guides/sveltekit/
  `);
```